src/components/videosList/videosContainer.css

```diff
@@ -0,0 +1,15 @@
+.videos {
+  flex: 0.9;
+  background-color: #f9f9f9;
+  padding: 40px 20px;
+  padding-bottom: 0;
+  outline: 1px solid red;
+}
+
+.resize-container {
+  flex: 1;
+  background-color: #f9f9f9;
```

Comment:
assigning a CSS-variable for the repeated color property will be much helpful incase of theme changing(background-color here)